Userland/Libraries/LibSoftGPU/Clipper.cpp

```diff
@@ -59,26 +40,14 @@ void Clipper::clip_triangle_against_frustum(Vector<Vertex>& input_verts)
     auto read_from = &list_a;
     auto write_to = &list_b;
 
-    for (size_t plane = 0; plane < NUMBER_OF_CLIPPING_PLANES; plane++) {
-        write_to->clear_with_capacity();
-        // Save me, C++23
-        for (size_t i = 0; i < read_from->size(); i++) {
-            const auto& curr_vec = read_from->at((i + 1) % read_from->size());
-            const auto& prev_vec = read_from->at(i);
-
-            if (point_within_clip_plane(curr_vec.clip_coordinates, static_cast<ClipPlane>(plane))) {
-                if (!point_within_clip_plane(prev_vec.clip_coordinates, static_cast<ClipPlane>(plane))) {
-                    auto intersect = clip_intersection_point(prev_vec, curr_vec, static_cast<ClipPlane>(plane));
-                    write_to->append(intersect);
-                }
-                write_to->append(curr_vec);
-            } else if (point_within_clip_plane(prev_vec.clip_coordinates, static_cast<ClipPlane>(plane))) {
-                auto intersect = clip_intersection_point(prev_vec, curr_vec, static_cast<ClipPlane>(plane));
-                write_to->append(intersect);
-            }
-        }
-        swap(write_to, read_from);
-    }
+    // Save me, C++23. With enum reflection it will be possible to loop
+    // over all the available enum values.
```

Comment:
This comment is a bit weird; I think we should either leave it out or make it a C++23 FIXME

Reply:
I only extended the comment. Can make it a FIXME though.